ml_mr/estimation/quantile_iv.py

```diff
@@ -290,6 +292,171 @@ def from_results(cls, dir_name: str) -> "QuantileIVEstimator":
         return cls(exposure_network, outcome_network, meta=meta, covars=covars)
 
 
+class QuantileIVLinearEstimator(MREstimatorWithUncertainty):
+    """
+    QuantileIV estimator that uses the linear inference strategy, which
+    consists of performing IV linear regression manually in the new
+    representation of the exposure learned by the outcome neural network.
+    """
+    def __init__(
+        self,
+        dataset: IVDataset,
+        parent: QuantileIVEstimator
+    ):
+        self.parent = parent
+        self.exposure_network = parent.exposure_network
+        self.outcome_network = parent.outcome_network
```

Comment:
Unneeded. I think it will be less error prone to index from the parent directly.